### PR TITLE
Add analysis example for Bu+Bc search

### PR DESCRIPTION
**Pull request by zuoxunwu**
#### PR content:
Add an example for Bu+Bc search, which is a spin-off of the Bc2TauNu search. This folder is setup to host future tests and developments while keeping the Bc example intact.

Current steps in this example are taken from the Bc2TauNu steps with minor modifications.

#### Test info:
Runs well in local tests.

#### Related PR:
Not related to previous PRs.

### CODE REVIEW

File: examples/FCCee/flavour/Bc2TauNu/analysis_stage2.py [hunk 1]
```diff
@@ -27,6 +27,12 @@
 
 ROOT.gInterpreter.ProcessLine('''
 TMVA::Experimental::RBDT<> bdt("Bc2TauNu_BDT", "/eos/experiment/fcc/ee/analyses/case-studies/flavour/Bc2TauNu/xgb_bdt_vtx.root");
+computeModelBc = TMVA::Experimental::Compute<18, float>(bdt);
+''')
+
+
+ROOT.gInterpreter.ProcessLine('''
+TMVA::Experimental::RBDT<> bdt("Bu2TauNu_BDT", "/afs/cern.ch/work/x/xzuo/public/FCC_files/Bc2TauNu/data/ROOT/xgb_bdt_Bu2TauNu_vtx.root");
 computeModelBis = TMVA::Experimental::Compute<18, float>(bdt);
```
Comment (clementhelsens) [inline examples/FCCee/flavour/Bc2TauNu/analysis_stage2.py:36]: could you remove this before I merge, such that the Bc2TauNu stays the same as before?

Comment (clementhelsens) [reply examples/FCCee/flavour/Bc2TauNu/analysis_stage2.py:36]: I put it here, but there are other changes in Bc2TauNu that you should revert.

Comment (zuoxunwu) [reply examples/FCCee/flavour/Bc2TauNu/analysis_stage2.py]: Now it is reverted.

**Comment by zuoxunwu** [comment]
Hi Clement,
Changes to Bc2TauNu are reverted. Should be good to merge now.
Best,
Xunwu

**Comment by clementhelsens** [comment]
yes, thanks, will merge now

**Pull request merged**